custom_components/abbfreeathome_ci/binary_sensor.py

```diff
@@ -97,6 +109,11 @@ def __init__(
         self._value_attribute = value_attribute
         self._sysap_serial_number = sysap_serial_number
 
+        # If the channel name is different from the device name, it's likely
+        # a dedicated sensor with it's own naming convention. Use it's channel name instead.
+        if device.channel_name != device.device_name:
+            entity_description_kwargs.pop("translation_key")
```

Comment:
After I added the additional if
```
if 'translation_key' in entity_description_kwargs.keys():
    entity_description_kwargs.pop("translation_key")
```
it is working like a charme

Reply:
Ok, a bit odd. All should have the key. I can add the check just to be sure though.

Reply:
I also don't understand it after checking your code ...